### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by stweil**
They were found using codespell.

Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: qt/src/OutputTextEdit.cc [hunk 1]
```diff
@@ -217,7 +217,7 @@ void OutputTextEdit::paintEvent(QPaintEvent* e) {
 			rect.setLeft(startLine.cursorToX(startLinePos) - 0.5);
 			painter.drawRect(rect);
 
-			// Draw selections inbetween
+			// Draw selections between
```
Comment (manisandro) [inline qt/src/OutputTextEdit.cc]: This one was actually correct.

Comment (stweil) [reply qt/src/OutputTextEdit.cc]: I'm not a native speaker, but all information which I can find says it's wrong (as does codespell). See for example https://www.grammarly.com/blog/in-between-or-inbetween/.

Comment (stweil) [reply qt/src/OutputTextEdit.cc]: Do you want to keep it nevertheless?`Or replace it by in-between? Then I can update the commit.

Comment (manisandro) [reply qt/src/OutputTextEdit.cc]: Ah right, should be *in between*.

**Comment by stweil** [comment]
Updated the pull request to use _in between_ and added another commit which fixes spelling of a variable name.

**Comment by manisandro** [comment]
Thanks!

**Pull request merged**